app/upgrades/v12/upgrades.go

```diff
@@ -119,10 +128,29 @@ func CreateV12UpgradeHandler(
 			panic("mm.Modules[icatypes.ModuleName] is not of type ica.AppModule")
 		}
 		icamodule.InitModule(ctx, controllerParams, hostParams)
+		logger.Info("upgraded ica module")
 
 		// IBCFee
 		vm[ibcfeetypes.ModuleName] = mm.Modules[ibcfeetypes.ModuleName].ConsensusVersion()
+		logger.Info(fmt.Sprintf("ibcfee module version %s set", fmt.Sprint(vm[ibcfeetypes.ModuleName])))
+
+		// Run migrations
+		versionMap, err := mm.RunMigrations(ctx, cfg, vm)
+
```

Comment:
we should handle err here.

Reply:
We return the err and It’s handled up the stack. This is done just so we can run the globalfee after. (Since before we just did "return RunMigrations()"